src/System.Reflection.Metadata/ref/System.Reflection.Metadata.cs

```diff
@@ -99,9 +99,7 @@ namespace System.Reflection.Metadata
         public System.Reflection.Metadata.StringHandle Name { get { throw null; } }
         public System.Reflection.Metadata.BlobHandle PublicKey { get { throw null; } }
         public System.Version Version { get { throw null; } }
-#if !NETSTANDARD11
```

Comment:
You need to sanitize the changes so only include the necessary changes you want.

Keep the ifdefs.